### PR TITLE
navigate after login

### PR DESCRIPTION
**Pull request by Quken**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,37 +1,51 @@
...
+                </li>
+              )}
+            </ul>
+          </nav>
+
+          <Switch>
+            <Route exact path='/'>
+              <CarouselView />
+            </Route>
+            <Route path='/login' component={Login} />
```
Comment (Quken) [inline client/src/App.js]: Here was 
`<Route path='/login' >
    <Login />
</Route>
`

File: client/src/components/menu.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Login } from './login';
+
+export const MenuView = () => {
+  const [user, setUser] = React.useState();
+
+  React.useEffect(() => {
```
Comment (ann-maier) [inline client/src/components/menu.jsx]: componentDidMount

File: client/src/components/menu.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <nav>
+      <ul>
+        <li>
+          <Link to='/'>Home</Link>
+        </li>
+        {user ? (
+          <h1>Welcome, {user}</h1>
+        ) : (
+            <li>
+              <Login setUser={user => setUser(user)} />
```
Comment (ann-maier) [inline client/src/components/menu.jsx]: pass `setState` to invoke re-render of the menu component.

File: client/src/App.js (hunk 1 above)
Comment (ann-maier) [reply client/src/App.js]: good that you've removed that shit 👍

File: client/src/components/auth.jsx [hunk 4]
```diff
@@ -4,49 +4,74 @@ import { Button, Col, Container, Form, Row } from 'react-bootstrap';
 import { post } from '../utils/httpService';
 import { LOGIN_URL } from '../utils/constants';
 
-export const Auth = (props) => {
-  const [login, setLogin] = React.useState('');
-  const [password, setPassword] = React.useState('');
+export class Auth extends React.Component {
```
Comment (ann-maier) [inline client/src/components/auth.jsx]: no need to create class component, you can revert that.

**Pull request merged**